src/marks/text.js

```diff
@@ -39,7 +39,10 @@ export class Text extends Mark {
         {name: "rotate", value: numberChannel(vrotate), optional: true},
         {name: "text", value: text, filter: nonempty}
       ],
-      options,
+      {
+        ...options.stroke && {paintOrder: "stroke", strokeWidth: 3},
```

Comment:
The strokeWidth only has an effect if there is a stroke, so we can simply move this to up the `defaults` declaration. Likewise, we could move the `paintOrder` up to `defaults` too and add some logic to style.js so only apply the default `paintOrder` if there’s both a fill and a stroke.

Alternatively, I was thinking that the strokeWidth would be 3 only if _both_ a fill and stroke are specified, and that it would be 1.5 (or 1?) if only the stroke were specified.